src/altnet.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_ALTNET_H
+#define BITCOIN_ALTNET_H
+
+#include <alt_processing.h>
+#include <drivers.h>
+#include <net.h>
+#include <netmessagemaker.h>
```

Comment:
I think this should be way more disconnected from the existing P2P stack. Accessing anything but validation.h and some utils seems bad.